prestoadmin/util/presto_config.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import os
+from StringIO import StringIO
+
+from fabric.context_managers import settings
+from fabric.operations import get
+from fabric.state import env
+from prestoadmin.config import get_conf_from_properties_data
+from prestoadmin.util.constants import REMOTE_CONF_DIR, CONFIG_PROPERTIES
+
+HTTP_ENABLED_KEY = 'http-server.http.enabled'
+HTTPS_ENABLED_KEY = 'http-server.https.enabled'
+HTTP_PORT_KEY = 'http-server.http.port'
+HTTPS_PORT_KEY = 'http-server.https.port'
+CLIENT_KEYSTORE_PATH_KEY = 'http-server.https.keystore.path'
```

Comment:
Per offline discussion, we need to make this match whatever is on the upcoming TD release.
If it changes when this lands on FB master, we'll have yet another place where we have a version dependency.